capstone-frontend/src/components/LeftSideBar.jsx

```diff
@@ -177,7 +177,7 @@ export const LeftSideBar = withRouter((props) => {
 
   const classes = useStyles();
   const theme = useTheme();
-  const [open, setOpen] = React.useState(false);
+  const [open, setOpen] = React.useState(true);
   const [openList, setOpenList] = React.useState(true);
```

Comment:
Could we instead leave it as false initially?

Reply:
Ok